canvas_modules/harness/src/client/components/sidepanel-properties.jsx

```diff
@@ -569,6 +601,43 @@ export default class SidePanelModal extends React.Component {
 			</Button>
 		</div>);
 
+		const setStaticRowsPropertyId = (
+			<div className="harness-sidepanel-children" id="sidepanel-properties-set-static-rows-propertyid">
+				<TextInput
+					labelText="Set the propertyId for setting static rows"
+					id="harness-propertyId-staticRows"
+					placeholder='{ "name": "parameterName" }'
+					invalid={this.state.invalidSetStaticRowPropertyId}
+					invalidText="Please enter valid JSON"
+					onChange={ this.setStaticRowsPropertyId }
+					helperText='PropertyId format: {"name": "unique_id_for_control"}'
+				/>
+			</div>
+		);
+
+		const setStaticRowsIndexes = (
+			<div className="harness-sidepanel-children" id="sidepanel-properties-set-static-rows-indexes">
+				<TextInput
+					labelText="Set the indexes for static rows"
+					id="harness-propertyId-staticRows"
```

Comment:
Can you give these unique ids?
<img width="663" alt="image" src="https://user-images.githubusercontent.com/1628100/129262212-77a91dee-1046-49e5-b1e3-a2959b9594c5.png">